files/build_templates/docker_image_ctl.j2

```diff
@@ -261,10 +322,15 @@ stop() {
 
 OP=$1
 DEV=$2 # namespace/device number to operate on
+NAMESPACE_PREFIX="asic"
 if [ "$DEV" ]; then
-    NET_NS="asic$DEV" #name of the network namespace
-else
+    NET_NS="$NAMESPACE_PREFIX$DEV" #name of the network namespace
+    SONIC_CFGGEN="sudo sonic-cfggen -n $NET_NS"
+    SONIC_DB_CLI="sudo sonic-db-cli -n $NET_NS"
```

Comment:
>sudo  [](start = 18, length = 5)

Please check why you need `sudo`.

Reply:
I did a check if we need sudo, and Yes we need sudo as we start some services as admin. This is the [User] definition in the service files.

acms.service:User=admin
bgp.service:User=admin
bgp@.service:User=admin
database.service:User=root
database@.service:User=root
dhcp_relay.service:User=admin
lldp.service:User=admin
lldp@.service:User=admin
pmon.service:User=admin
radv.service:User=admin
swss.service:User=root
swss@.service:User=root
syncd.service:User=root
syncd@.service:User=root
teamd.service:User=admin
teamd@.service:User=admin